### ISSUE TITLE
Migrate CRD v1beta1 to v1

### ISSUE THREAD

**Issue by rodcloutier** [pulumi/pulumi-kubernetes-operator#99]
### Problem description

CRD `v1beta1` are deprecated and `v1` should be used.

Upon install we get the following warning :
`Warning: apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition`

Simply changing to `v1` gives the following error:
```
error: error validating "deploy/crds/pulumi.com_stacks.yaml": error validating data: [ValidationError(CustomResourceDefinition.spec): unknown field "subresources" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1.CustomResourceDefinitionSpec, ValidationError(CustomResourceDefinition.spec): unknown field "validation" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1.CustomResourceDefinitionSpec, ValidationError(CustomResourceDefinition.spec): unknown field "version" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1.CustomResourceDefinitionSpec]; if you choose to ignore these errors, turn validation off with --validate=false```
```

**Comment by viveklak** [comment]
Note that CRD v1.beta1 will be removed in 1.22 - about 6 months from now.

**Comment by viveklak** [comment]
Fixed in #127
